### PR TITLE
Update env sample

### PR DESCRIPTION
**Pull request by tai271828**
Issue #35 . See detailed descriptions in the commit messages.

### CODE REVIEW

File: backend/.env.sample [hunk 1]
```diff
@@ -21,12 +22,16 @@ GOOGLE_MAP_API_KEY=
...
+OHSHOWN_FRONTEND_DOMAIN="http://localhost/"
+#OHSHOWN_FRONTEND_DOMAIN="http://ohshown.site/"
 
 DISFACTORY_PGDATA_PATH=/tmp/disfactory_data/
 DISFACTORY_PGDATA_BACKUP_PATH=/tmp/disfactory_db_dump/
 
 DISFACTORY_CADDY_STATIC_DIR=/tmp/disfactory/static
 
-COMPOSE_PROJECT_NAME=staging
+COMPOSE_PROJECT_NAME=localdev
```
Comment (tai271828) [inline backend/.env.sample:35]: In my opinion, `localdev` is a better name for `staging` when developing locally.

File: backend/.env.sample [hunk 2]
```diff
@@ -21,12 +22,16 @@ GOOGLE_MAP_API_KEY=
 # will be deprecated
 DISFACTORY_IMGUR_CLIENT_ID=your_imgur_id
 DISFACTORY_BACKEND_MEDIA_ROOT="./images/"
-DISFACTORY_BACKEND_DOMAIN="https://api.disfactory.tw/"
-DISFACTORY_FRONTEND_DOMAIN="https://disfactory.tw/"
+DISFACTORY_BACKEND_DOMAIN="http://localhost/"
```
Comment (EagleC0318) [inline backend/.env.sample:25]: Why the prefix of backend domain is DISFACTORY but the frontend one is OHSHOWN?

**Comment by EagleC0318** [comment]
Just wondering, what's the actual impact of this PR?

**Review by EagleC0318 — APPROVED** [review]

**Comment by tai271828** [comment]
Discussed with eagle offline already.

**Pull request merged**